etc/kayobe/environments/aufn-ceph/kolla/inventory/overcloud-services.j2

```diff
@@ -416,9 +416,6 @@ monitoring
 [prometheus-libvirt-exporter:children]
 compute
 
-[prometheus-msteams:children]
-prometheus-alertmanager
```

Comment:
Can we document this in the upgrade guide, in case customers are still unaware that teams integration is no more